### PR TITLE
[14_0_X] Apply MUDPGNANO EventContent customizations to EDMNANO and NANOSIM as well

### PR DESCRIPTION
**Pull request by battibass**
#### PR description:

This is a **backport of PR #44847**.

It implements a bugfix, basted on the results of the first tentative of automating the production of the MUDPG custom NANO ntuples described in [this JIRA ticket](https://its.cern.ch/jira/browse/PDMVRERECO-83), intended to be deployed in production this year.

#### PR validation:

No changes expected in `runTheMartrix.py` workflows `-w nano -l 2500.312,2500.313`.

Change in event content if the `cmsDriver.py` command picked [here](https://cms-pdmv-prod.web.cern.ch/rereco/api/requests/get_cmsdriver/ReReco-Run2024B-Muon0-ZMu_PromptMUODPGNano-00001) is ran under `CMSSW_14_0_X`.

Were both verified.

### CODE REVIEW

File: DPGAnalysis/MuonTools/python/muNtupleProducerBkg_cff.py [hunk 1]
```diff
@@ -11,8 +11,9 @@
 
 def muDPGNanoBkgCustomize(process) :
 
-     if hasattr(process, "NANOAODoutput"):
-          process.NANOAODoutput.outputCommands.append("keep nanoaodFlatTable_*Table*_*_*")
-          process.NANOAODoutput.outputCommands.append("drop edmTriggerResults_*_*_*")
+     for output in ["NANOEDMAODoutput", "NANOAODoutput", "NANOAODSIMoutput"]:
+          if hasattr(process, output):
+               getattr(process,output).outputCommands.append("keep nanoaodFlatTable_*Table*_*_*")
```
[the remark this answers is not in the document]
Comment (hqucms) [reply DPGAnalysis/MuonTools/python/muNtupleProducerBkg_cff.py:16]: @vlimant You mean we change `keep nanoaodFlatTable_*Table_*_*` to `keep nanoaodFlatTable_*Table*_*_*` in [NanoAODOutputModule.cc](https://github.com/cms-sw/cmssw/blob/e2e3c528c6f6be69d9eeb2e88674b95bf56465c1/PhysicsTools/NanoAOD/plugins/NanoAODOutputModule.cc#L425)?

[the remark this answers is not in the document]
Comment (vlimant) [reply DPGAnalysis/MuonTools/python/muNtupleProducerBkg_cff.py:16]: there and there https://github.com/cms-sw/cmssw/blob/e2e3c528c6f6be69d9eeb2e88674b95bf56465c1/PhysicsTools/NanoAOD/python/NanoAODEDMEventContent_cff.py#L6

**Comment by hqucms** [comment]
enable nano

**Comment by hqucms** [comment]
please test

**Comment by hqucms** [comment]
+1

**Comment by rappoccio** [comment]
+1

**Pull request merged**